refactor(cdr-radio and cdr-label-wrapper): refactor cdr-radio and cdr…

KenjiCrosland opened the pull request:
…-label-wrapper for Vue 3

Used computed v-model for cdr-radio. Included more examples to cdr-radio kitchen sink page. Added a
radio form group with validation on the formgroup example page. Updated how styles are concatenated
for CdrLabelWrapper

CDR-2503

bonpixel commented:
ApPrOvE

Commit:
refactor(cdr-radio and cdr-label-wrapper): refactor cdr-radio and cdr-label-wrapper for Vue 3

Used computed v-model for cdr-radio. Included more examples to cdr-radio kitchen sink page. Added a
radio form group with validation on the formgroup example page. Updated how styles are concatenated
for CdrLabelWrapper

CDR-2503

mhewson reviewed (commented):
@KenjiCrosland can you take a look at the couple of WCAG issues and see if these are real or not?

mhewson reviewed (commented):
@KenjiCrosland can you take a look at the couple of WCAG issues and see if these are real or not?

Commit:
refactor(cdr-label-standalone, cdr-label-wrapper): accessibility changes

Modify required * span to use aria-hidden=true. Remove hover styling from disabled form fields

Commit:
Fix snapshot
